crates/engine/tree/src/persistence.rs

```diff
@@ -12,13 +12,14 @@ use tokio::{
 };
 
 /// Writes parts of reth's in memory tree state to the database.
+///
+/// It's expected that this will be spawned in its own thread with [`std::thread::spawn`], since
+/// this performs blocking database operations.
 pub struct Persistence<DB> {
```

Comment:
I'd also like a `PersistenceHandle` type that wraps the corresponding sender

Reply:
added one